src/repos/NotificationRepo.ts

```diff
@@ -122,8 +203,25 @@ const notify = async (articleIDs: string[]): Promise<void> => {
   });
 };
 
+/**
+ * Send notifications for new articles have been posted by publications.
+ */
+const notifyWeeklyDebrief = async (users: User[]): Promise<void> => {
+  users.forEach(async (user) => {
+    if (user.deviceType === ANDROID) {
+      sendWeeklyDebriefAndroidNotification(user);
+    }
+    if (user.deviceType === IOS) {
```

Comment:
else if, you can't ever have both of these cases so it doesn't make sense to check if for both